### ISSUE TITLE
bug: Multi-modal support breaks adding dataset items from traces

### ISSUE THREAD

**Issue by nitin302** [langfuse/langfuse#4609]
### Describe the bug

When we enable multi-modal support - https://langfuse.com/docs/tracing-features/multi-modality, There is a change in input visible in the Langfuse generation.

For example:

`[
    {
        "text": "What’s in this image?",
        "type": "text"
    },
    {
        "type": "image_url",
        "image_url": {
            "url": "@@@langfuseMedia:type=image/jpeg|id=38a8259f-79f9-451a-8f8b-949e12bf52ed|source=base64_data_uri@@@"
        }
    }
]`

This breaks the https://langfuse.com/docs/datasets/overview - "Add Items from Langfuse Trace" feature, as now we are unable to directly add it to a dataset and use that for experiment.

Expected Flow:

Even when multi-modal support is enabled, when adding a generation as a dataset item, it should automatically convert the image_url to base64 ... preserving its original input in the item.

### To reproduce

1. Enable multi-modal support
2. Trace any LLM call with images as base64 URI
3. Add generation to data item
4. Run Experiment with this data item
5. LLM call returns error as the image image_url is invalid

### Are you interested to contribute a fix for this bug?

No

**Comment by hassiebp** [comment]
@nitin302 - thanks for raising this! To clarify, you'd like to see the following: When selecting a multimodal generation in the trace detail view and clicking `Add to dataset` you'd like to have the Dataset Item form prefilled with the generation and the original base64 data URI reinserted?

**Comment by nitin302** [comment]
yes correct @hassiebp ... this would ensure we can add items to dataset even with multi-modal input seamlessly to run experiments.

**Comment by nitin302** [comment]
any update please? @marliessophie @hassiebp

**Comment by hassiebp** [comment]
Hi @nitin302 - are you executing dataset runs locally or are you using the langfuse-hosted prompt experiments feature? In case you are running them locally, you may use the `resolveMediaReferences` utility method on the Langfuse client to reinsert the base64 data URI representation of your media asset. [Docs](https://langfuse.com/docs/tracing-features/multi-modality#3-resolving-media-references). 

If you are using prompt experiments, this is currently not supported, but please keep an eye on this issue to track progress on that.

**Comment by nitin302** [comment]
@hassiebp - thank you. I was able to use `resolveMediaReferences` successfully.
